CIP-0067/README.md

```diff
@@ -0,0 +1,53 @@
+---
+CIP:
+Title: Asset Name Label Registry
+Authors: Alessandro Konrad <alessandro.konrad@live.de>, Thomas Vellekoop <thomas.vellekoop@iohk.io>
+Comments-URI:
+Status: Draft
+Type: Informational
+Created: 2022-07-13
+Post-History:
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This proposal defines a standard to classify Cardano native assets by the asset name.
+
+## Motivation
+
+As more assets are minted and different standards emerge to query data for these assets, it's getting harder for 3rd parties to determine the asset type and how to proceed with it. This standard is similar to [CIP-0010](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0010), but focuses on the asset name of an asset.
+
+
+## Specification
+
+To classify assets the `asset_name` needs to be prefixed with an opening and closing parentheses and the label in between: `({Label})`.
```

Comment:
Alternatives to parentheses to make it more obscure:
- Single colon, e.g. `:222:`
- Double colon, e.g. `::222::`

Reply:
Or, what about prefixing every thing with `CIP67:{label}:` ? A bit longer, but it leaves 24-20 bytes for the asset name. One can also find a more compact binary encoding which would take less bytes but loose the "readability" aspect of it. 

For example, consider the prefix to be `0xC067`, that's only 2 bytes and reads well once hex-encoded.